jstp/src/main/java/com/metarhia/jstp/connection/SessionRestorationPolicy.java

```diff
@@ -39,7 +39,7 @@ public void onTransportAvailable(Connection connection, String appName, String s
   /**
    * Checks if reconnection needed when transport is ready
```

Comment:
Can you also add: 'if true then this policy will try to reconnect to the remote side after underlying transport has established a connection' to the description